From b2fc7224731a1b422da36e085e2a47ea50f70582 Mon Sep 17 00:00:00 2001
From: Minh Le <minh.le.aj@renesas.com>
Date: Mon, 21 Apr 2025 14:56:26 +0700
Subject: [PATCH 17/52] i2c: rcar_i2c: Add R-Car Gen5 support

I2C on R-Car Gen5 is treated the same as Gen3 and Gen4, and shares the
same driver.

Signed-off-by: Minh Le <minh.le.aj@renesas.com>
Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 drivers/i2c/rcar_i2c.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/i2c/rcar_i2c.c b/drivers/i2c/rcar_i2c.c
index 3bd5108fd23..9562119bb5b 100644
--- a/drivers/i2c/rcar_i2c.c
+++ b/drivers/i2c/rcar_i2c.c
@@ -368,6 +368,7 @@ static const struct udevice_id rcar_i2c_ids[] = {
 	{ .compatible = "renesas,rcar-gen2-i2c", .data = RCAR_I2C_TYPE_GEN2 },
 	{ .compatible = "renesas,rcar-gen3-i2c", .data = RCAR_I2C_TYPE_GEN3 },
 	{ .compatible = "renesas,rcar-gen4-i2c", .data = RCAR_I2C_TYPE_GEN3 },
+	{ .compatible = "renesas,rcar-gen5-i2c", .data = RCAR_I2C_TYPE_GEN3 },
 	{ }
 };
 
-- 
2.34.1

